components/sendbird/sendbird.app.mjs

```diff
@@ -1,11 +1,233 @@
+      options: constants.CHANNEL_TYPE_OPTS,
+    },
+    applicationId: {
+      type: "string",
+      label: "Application Id",
+      description: "Specifies the unique ID of the Application, To find the application ID, sign in to [Sendbird Dashboard](https://dashboard.sendbird.com/auth/signin), go to **Settings > Application > General**, and then check Application ID.",
+    },
+    channelUrl: {
+      type: "string",
+      label: "Channel URL",
+      description: "Specifies the url of the channel.",
+      async options({ applicationId }) {
+        if (!applicationId) {
+          return [];
+        }
+        const { channels } = await this.listChannels(applicationId);
+        return channels.map((channel) => ({
+          label: channel.name,
+          value: channel.channel_url,
+        }));
+      },
+    },
+    messageTs: {
+      type: "integer",
+      label: "Message Timestamp",
+      description: "Specifies the timestamp to be the reference point of the query in [Unix milliseconds](https://en.wikipedia.org/wiki/Unix_time). Either this or the `Message Id` should be specified.",
+      optional: true,
+    },
+    messageId: {
+      type: "integer",
+      label: "Message Id",
+      description: "Specifies the unique ID of the message to be the reference point of the query. Either this or the `Message Timestamp` should be specified.",
+      optional: true,
+    },
+    prevLimit: {
+      type: "integer",
+      label: "Prev Limit",
+      max: 200,
+      min: 0,
+      description: "Specifies the number of previously sent messages to retrieve before `message_ts`. For example, if `message_ts=1484202848298`, then `prev_limit=50` returns 50 messages sent by 1484202848297 (message_ts - 1).",
+      optional: true,
+    },
+    nextLimit: {
+      type: "integer",
+      label: "Next Limit",
+      max: 200,
+      min: 0,
+      description: "Specifies the number of sent messages to retrieve after `message_ts`. For example, if `message_ts=1484202848298`, then `next_limit=50` returns 50 messages sent from 1484202848299 (message_ts + 1).",
+      optional: true,
+    },
+    include: {
+      type: "boolean",
+      label: "Include",
+      description: "Determines whether to include messages sent exactly on the specified `message_ts` in the results. (Default: **true**)",
+      optional: true,
+    },
+    reverse: {
+      type: "boolean",
+      label: "Reverse",
+      description: "Determines whether to sort the results in reverse chronological order. If set to **true**, messages appear in reverse chronological order where the newest comes first and the oldest last. (Default: **false**)",
+      optional: true,
+    },
+    senderIds: {
+      type: "string[]",
+      label: "Sender Ids",
+      description: "Restricts the search scope to only retrieve messages sent by one or more users with the specified IDs listed in a comma-separated string.",
```

Comment:
The description suggests this is a comma-separated string, but we're presenting the user with (one or more) options to select.

Also, I noticed it's passed as-is, rather than converted from an array to this comma-separated string (with `Array.join()` or a similar method), but perhaps either the SDK or their API handles this?

Regardless, I think the description should match the user's experience, for example:
```suggestion
      description: "Restricts the search scope to only retrieve messages sent by one or more users selected from the list.",
```